polygon-validator-signer-balance-bot/src/agent.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { FindingType, FindingSeverity, Finding, getJsonRpcUrl, ethers, HandleBlock, Initialize } from "forta-agent";
+import { provideHandleBlock, provideInitialize } from "./agent";
+import { TestBlockEvent } from "forta-agent-tools/lib/test/index";
+import { Flag, POLYGON_VALIDATOR_SIGNER_ADDRESS, MINIMUM_THRESHOLD } from "./constants";
+
+describe("POLYGON-VALIDATOR-SIGNER BOT TEST SUITE", () => {
+  let handleBlock: HandleBlock;
+  let mockEthersProvider: ethers.providers.JsonRpcProvider;
+  let initialize: Initialize;
+  let flag: Flag;
+  jest.setTimeout(1000000000);
+
+  beforeEach(async () => {
+    mockEthersProvider = new ethers.providers.JsonRpcProvider(getJsonRpcUrl());
```

Comment:
Right now, this test suite is not using a mock `provider` and thus is actually making network calls.

Please use the `MockEthersProvider` from the `forta-agent-tools` library. Section in the library can be found [here](https://github.com/NethermindEth/general-agents-module#mockethersprovider). An example usage can be found [here](https://github.com/NethermindEth/Forta-Agents/blob/main/DYDX-Bots/Large-USDC-Staking-Withdrawals/src/agent.spec.ts#L28) and [here](https://github.com/NethermindEth/Forta-Agents/blob/main/DYDX-Bots/Large-USDC-Staking-Withdrawals/src/agent.spec.ts#L37-L42).